runbotics/runbotics-orchestrator-ui/src/src-app/components/cron/utils.ts

```diff
@@ -71,3 +71,5 @@ export function usePrevious(value) {
 
     return ref.current;
 }
+
+export const checkIsShortcutsArray = (shortcuts: any): shortcuts is ShortcutsType => Array.isArray(shortcuts);
```

Comment:
kinda nothing specific for shortcuts `isArray` would be sufficient

Reply:
done